docs/css/fui-site.css

```diff
@@ -4189,6 +4189,7 @@ This will render 4 boxes spanning 2 cols each indented 2 cols
     --fd-button-border-color: transparent;
     --fd-button-background-color: hsl(217, 21%, 21%); }
 
+/* stylelint-disable no-duplicate-selectors */
```

Comment:
I think this PR should be held until #1345 is merged.  The files `fui-site.css` and `fiori-fundamentals.css` are no longer required to be updated in source control (since they are generated).

Reply:
#1345 has been merged. ;)